api/schema/application_azure_resources_resource.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Illumio, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	resource_schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var (
+	applicationAzureResourcesResource = Resource{
+		TypeName: "application_azure_resources",
+		Schema: resource_schema.Schema{
+			Version:     1,
+			Description: "Manages a set of Azure resources belonging to a single Azure account that are associated with a CloudSecure application.",
+			Attributes: map[string]resource_schema.Attribute{
+				IDFieldName: idAttribute,
+				"application_id": StringResourceAttributeWithMode{
+					StringAttribute: resource_schema.StringAttribute{
+						Description: "ID of the CloudSecure application.",
+						Required:    true,
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.RequiresReplace(),
+						},
+					},
+					attributeWithMode: attributeWithMode{
+						Mode: ImmutableAttributeMode,
+					},
+				},
+				"azure_resource_ids": resource_schema.ListAttribute{
```

Comment:
Since you now have `azure` in the resource name, it is redundant as attribute name prefix.
```suggestion
				"resource_ids": resource_schema.ListAttribute{
```